docs/wiki/[User-Guide]-Deploy-Management-Resources.md

```diff
@@ -1 +1,141 @@
-_coming soon_
\ No newline at end of file
+## Overview
+This example will deploy resources for centrally managing the Enterprise Scale Landing Zone. This module will deploy the following resources with default configurations that can be adjusted once you are up and going:
+
+
+   * resource group
+   * log analytics workspace
+   * automation account 
+   * linked service to the analytics automation account
+
+in the subscription of your specification.  
+
+The module then goes onto configure Deploy if Not Exist policies to enable features within Log Analytics and Sentinel including
+but not limited to: 
```

Comment:
I think this should either be a:
- Complete list of policies amended (preference)
- OR
- None of them at all and just say policies are amended